maestro-android/src/androidTest/java/dev/mobile/maestro/MaestroDriverService.kt

```diff
@@ -287,6 +301,29 @@ class Service(
         responseObserver.onCompleted()
     }
 
+    override fun fetchSession(
+        request: MaestroAndroid.SessionRequest,
+        responseObserver: StreamObserver<MaestroAndroid.SessionResponse>
+    ) {
+        val sessionReq = Request.Builder()
+            .get()
+            .url("http://localhost:7008/session")
+            .build()
+
+        val sessionInfo = httpClient
```

Comment:
This call might throw an error, you need to propagate it properly to `responseObserver`